mapcreator/building.py

```diff
@@ -245,29 +251,26 @@ def __str__(self):
                 lines.extend(self.output.getvalue().split('\n'))
         return '\n'.join(lines)
 
+# Outputs only one combined file
 def process_satellite_with_gdal(satellitestatus, debug = False):
-    outpath = path.join(FINALIZED_DIR, INTERMEDIATE_SATELLITE_FORMAT.format(0))
-    
-    # Check which satellite files lie in the window and call GDAL with only them as input files
-    for satfile in satellitestatus.original_files:
-        gdal_info = gdal_util.Gdalinfo.for_file(satfile)
-        if satellitestatus.state.get_window_string_lowerleft_topright_cut(gdal_info):
-            satellitestatus.add_file_in_window(satfile)
-    if satellitestatus.files_in_window:
-        satfiles = ' '.join(satellitestatus.files_in_window)
+    if satellitestatus.current_files:
+        outpath = path.join(FINALIZED_DIR, INTERMEDIATE_SATELLITE_FORMAT.format(0))
+        satfiles = ' '.join(satellitestatus.current_files)
         projection_window = satellitestatus.state.get_window_string_lowerleft_topright()
         command = 'gdalwarp -tr {} {} -te_srs {} -t_srs {} -r bilinear -te {} {} {}'.format(OUTPUT_CELLSIZE, OUTPUT_CELLSIZE, LATLON_DATUM_IDENTIFIER, PROJECTION_IDENTIFIER, projection_window, satfiles, outpath)
+        if satellitestatus.state.has_satellite_system():
+            command = '{} -s_srs {} {}'.format(command[:8], satellitestatus.state.satellite_coordinatesystem, command[9:])
```

Comment:
Same thing here as in the earlier extend-command-by-slicing-string; I think refactoring these to use keyword arguments would really be the best idea at this point